unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/SettingsPanelHUD/Scripts/ControlsModule/SettingsControlController.cs

```diff
@@ -0,0 +1,13 @@
+using UnityEngine;
+
+namespace DCL.SettingsPanelHUD.Controls
+{
+    public abstract class SettingsControlController : ScriptableObject
```

Comment:
```suggestion
    public abstract class SettingsControlController : ScriptableObject
```

What do you think on renaming the `Control` concept?

The `ControlController` moniker leads to confusing naming (on top of using `Controller` for every of our subsystems).

I propose `Widget` but maybe there's a better word for this?

Reply:
The "Widget" concept is already being used in this new architecture:
- **SettingsPanel** contains one or more **Sections**.
- A **Section** contains one or more **Widgets**.
- A **Widget** contains one or more **Controls**.

We can think other name, but the fact of currently change the name of the "Control" concept would provoke a lot of changes in code, prefabs names, folders, configuration assets... :(

If it is only because the names "Control" and "Controller" can be a bit confusing, what do you think about to change only the class name by "SettingsControlBaseController" (this class is used as base of each specific control) and keep the "Control" concept in the system?

Reply:
Ok! No problem, it was just a minor detail. Also I didn't know we already were using the widgets concept. Maybe we can polish this down the road in another PR if we find it muddles the discussions. Thanks for the clarification!